draco/asp/helpers.lp

```diff
@@ -24,8 +26,28 @@ attribute((encoding,scale_type),E,T) :-
     attribute((encoding,channel),E,C),
     attribute(mark_scale_channel,M,T,C).
 
-% @helper(mark_channel_field) Whether a mark has an encoding for a field  and a channel.
+% @helper(mark_channel_field) Whether a mark has an encoding for a field and a channel.
 attribute(mark_channel_field,M,C,F) :-
     entity(encoding,M,E),
     attribute((encoding,channel),E,C),
     attribute((encoding,field),E,F).
+
+% @helper(mark_channel_scale_not_binned) Whether a mark has a scale type for a channel that is not binned.
+attribute(mark_channel_scale_not_binned,M,T,C) :-
```

Comment:
This seems like an odd helper. Why do we have a helper where you only get the scale when the encoding is not binned?

Reply:
I was trying to write the constraint for `bar_tick_continuous_x_y` (Bar and tick cannot have both x and y continuous.)
I think the prohibition for being continuous can be relaxed when the encoding is binned.

Reply:
Wouldn't it make more sense to call it `attribute(mark_channel_scale,M,T,C,B)` where B is true or false.

Reply:
I made some changes, but wasn't sure about the syntax.

Reply:
That looks great, don't you think?

Reply:
ohh I mean I wasn't sure the changes I made was the right way or not.

Reply:
There is no absolute right or wrong but it looks reasonable to me.